backend/src/main/java/com/woowacourse/levellog/levellog/dto/LevellogWithIdDto.java

```diff
@@ -0,0 +1,28 @@
+package com.woowacourse.levellog.levellog.dto;
+
+import com.woowacourse.levellog.levellog.domain.Levellog;
+import com.woowacourse.levellog.member.domain.Member;
+import com.woowacourse.levellog.team.domain.Team;
+import javax.validation.constraints.NotBlank;
```

Comment:
```suggestion
import com.woowacourse.levellog.levellog.domain.Levellog;
import javax.validation.constraints.NotBlank;
```
사용하지 않는 import 정리 부탁드립니다!